src/main/java/com/sap/sgs/phosphor/fosstars/data/github/InfoAboutVulnerabilities.java

```diff
@@ -37,11 +39,12 @@
    *
    * @param github An interface to the GitHub API.
    */
-  public InfoAboutVulnerabilities(GitHub github) throws IOException {
+  public InfoAboutVulnerabilities(GitHub github, NVD nvd) throws IOException {
```

Comment:
Maybe NVD should be initialized in this class, rather than being passed as an argument? What do you think?

Reply:
I think that the class needs to be configured with an instance of `NVD`.

Reply:
`NVD.download()` needs to be called before using the instance. This cause the class to download the data from NVD. We need to make sure that this is done once. Now, it's done by the demo tool, and it passes an initialized `NVD` instance to everyone who needs it.